service/tracker/consensus.go

```diff
@@ -51,7 +51,7 @@ func NewConsensus(log zerolog.Logger, db *badger.DB, hold RecordHolder) (*Consen
 	}
 
 	c := Consensus{
-		log:  log.With().Str("component", "consensus_tracker").Logger(),
+		log:  log.With().Str("engine", "consensus_tracker").Logger(),
```

Comment:
I still don't like the fact that this leaks to other packages since now suddenly they're all _engines_. I'd prefer to keep them as _components_.  If we really want to avoid a name conflict, we could label stuff as `engine_component` in the engine logs. Or some other better sounding name. Module perhaps?

Reply:
Once again this is an issue with the replace all, I missed it when I tried to fix it 😅 I'll have to take a look at every instance of `engine` in the codebase to make sure I didnt do it in more places. My bad